gramps/gen/utils/resourcepath.py

```diff
@@ -47,9 +67,9 @@ def __new__(cls):
     def __init__(self):
         if self.initialized:
             return
-        resource_file = os.path.join(os.path.abspath(os.path.dirname(
-            __file__)), 'resource-path')
-        installed = os.path.exists(resource_file)
+        package_path = os.path.join(os.path.abspath(os.path.dirname(
+            __file__)), '..', "..", "..")
```

Comment:
You need to normalize out the trailing `..`:
```Python
        package_path = os.path.normpath(os.path.join(os.path.abspath(os.path.dirname(
            __file__)), '..', "..", ".."))
```
Otherwise `tail` below is always `..`.

Reply:
Thanks.  According to the [documentation](https://docs.python.org/3/library/os.path.html) the `abspath` function returns a "normalized absolutized version of the pathname", so I just swapped the order of the `join` and `abspath` calls.  It should be fixed now.  I tested with a User scheme installation.